docs/deployments/api-configuration.md

```diff
@@ -16,6 +16,7 @@ Reference the section below which corresponds to your Predictor type: [Python](#
     path: <string>  # path to a python file with a PythonPredictor class definition, relative to the Cortex root (required)
     config: <string: value>  # arbitrary dictionary passed to the constructor of the Predictor (optional)
     python_path: <string>  # path to the root of your Python folder that will be appended to PYTHONPATH (default: folder containing cortex.yaml)
+    image: <string> # optional image to override the configured cluster serving image (e.g. cortexlabs/python-serve[-gpu])
```

Comment:
nit: "# docker image to use for the Predictor (default: cortexlabs/python-serve[-gpu])"

Reply:
Done!